file.go

```diff
@@ -76,44 +75,64 @@ func (f *File) SaveAs(name string) error {
 
 // Write provides a function to write to an io.Writer.
 func (f *File) Write(w io.Writer) error {
-	_, err := f.WriteTo(w)
-	return err
-}
-
-// WriteTo implements io.WriterTo to write the file.
-func (f *File) WriteTo(w io.Writer) (int64, error) {
```

Comment:
I don't see anything replacing the lost `WriteTo` method. Does this mean that excel.File no longer satisfies io.WriteTo?

Reply:
WriteTo returns int64 and err. 
In prev version, excelize write from bytes buffer to a file, so File object can get the length of total bytes writen `https://golang.org/pkg/bytes/#Buffer.WriteTo`
Currently, this PR is using zip.Writer so this method is nolonger support.

Reply:
The problem is that `WriteTo` is part of the public interface of this library, we can't remove it without releasing this under version 3, and I don't really see a good reason to remove the function. For the sake of this PR, you could even use the old behavior of copying it through a buffer. It would be less efficient due to the copy but a comment saying to favor using Write until WriteTo can be improved would be enough for consumers of the library to know how to make an efficient choice without breaking existing code.